src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js

```diff
@@ -16,7 +16,7 @@ function ChildProgressBar(props) {
         editable = props.editable;
 
   const intervalWidthDuration = interval.endPosition - interval.startPosition,
-        intervalWidth = intervalWidthDuration / duration * 400;
+        intervalWidth = intervalWidthDuration / duration * 100;
```

Comment:
Just wondering why it was 400 before?

Reply:
The bar width was `400px` total. So I kept the same math but instead made it out of `100` and changed the postfix from `px` to `%` 😄